webui/apps/docs/public/json_schema/v1/schemas/DocumentManager/DocumentManager.json

```diff
@@ -1,5 +1,6 @@
 {
   "additionalProperties": false,
+  "description": "Manages a collection of documents and provides search functionality. Automatically embeds and syncs documents (\nprovided by loader) into similarity memory where they can be searched.",
```

Comment:
same comment -- would it be correct and helpful to add the parenthetical: similarity (semantic) memory...